diff -urN libgee-0.8.package.orig/libs/libgee-0.8/Makefile package/libs/libgee-0.8/Makefile
--- libgee-0.8.package.orig/libs/libgee-0.8/Makefile	1970-01-01 01:00:00.000000000 +0100
+++ package/libs/libgee-0.8/Makefile	2013-12-16 14:51:32.712662603 +0100
@@ -0,0 +1,76 @@
+#
+# Copyright (C) 2013 Luca Dionisi aka Lukisi
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libgee-0.8
+PKG_VERSION:=0.12.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=libgee-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=@GNOME/libgee/0.12/
+PKG_MD5SUM:=d9965e1797d76775ae440230f30585f8
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/libgee-$(PKG_VERSION)
+PKG_INSTALL:=1
+PKG_BUILD_DEPENDS:=vala/host glib2
+
+TARGET_LDFLAGS+=\
+	-Wl,-rpath-link=$(STAGING_DIR)/usr/lib
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+
+define Package/libgee-0.8
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=GObject based collection library v0.8
+  URL:=http://live.gnome.org/Libgee
+  DEPENDS:=+glib2
+endef
+
+define Package/libgee-0.8/description
+libgee is a collection library providing GObject-based interfaces and classes
+for commonly used data structures.
+endef
+
+define Build/Compile
+	$(MAKE) -C $(PKG_BUILD_DIR) \
+		DESTDIR="$(PKG_INSTALL_DIR)" \
+		all install
+endef
+
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/usr/{lib/pkgconfig,include/gee-0.8}
+
+	$(CP) \
+		$(PKG_INSTALL_DIR)/usr/lib/libgee-0.8.{so*,la} \
+		$(1)/usr/lib/
+
+	$(INSTALL_DATA) \
+		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/*.pc \
+		$(1)/usr/lib/pkgconfig/
+
+	$(INSTALL_DATA) \
+		$(PKG_INSTALL_DIR)/usr/include/gee-0.8/gee.h \
+		$(1)/usr/include/gee-0.8/
+
+	$(INSTALL_DIR) $(1)/usr/share/vala/vapi
+
+	$(INSTALL_DATA) \
+		$(PKG_INSTALL_DIR)/usr/share/vala/vapi/gee-0.8.vapi \
+		$(1)/usr/share/vala/vapi
+endef
+
+define Package/libgee-0.8/install
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) \
+		$(PKG_INSTALL_DIR)/usr/lib/libgee-0.8.so* \
+		$(1)/usr/lib/
+endef
+
+$(eval $(call BuildPackage,libgee-0.8))
