diff -urN pthsem.feeds.orig/packages/libs/pthsem/Makefile feeds/packages/libs/pthsem/Makefile
--- pthsem.feeds.orig/packages/libs/pthsem/Makefile	2013-11-21 18:52:04.777364223 +0100
+++ feeds/packages/libs/pthsem/Makefile	2013-11-21 22:15:57.395656151 +0100
@@ -21,7 +21,7 @@
   CATEGORY:=Libraries
   TITLE:=GNU pth extended with semaphore support
   URL:=http://www.auto.tuwien.ac.at/~mkoegler/index.php/pth
-  DEPENDS:=@USE_EGLIBC @TARGET_x86
+  DEPENDS:=@USE_EGLIBC
 endef
 
 define Package/pthsem/description
@@ -42,18 +42,28 @@
 	$(SED) \
 		's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/usr,g' \
 		$(2)/bin/pthsem-config
+	ln -sf pthsem-config         $(2)/bin/pth-config
 
 	$(INSTALL_DIR) $(1)/usr/include
 	$(CP)   $(PKG_INSTALL_DIR)/usr/include/*.h \
 		$(1)/usr/include/
+	ln -sf pthsem.h              $(1)/usr/include/pth.h
 	$(INSTALL_DIR) $(1)/usr/lib
 	$(CP)   $(PKG_INSTALL_DIR)/usr/lib/libpthsem.{a,la,so*} \
 		$(1)/usr/lib/
+	ln -sf libpthsem.a           $(1)/usr/lib/libpth.a
+	ln -sf libpthsem.la          $(1)/usr/lib/libpth.la
+	ln -sf libpthsem.so.20.0.28  $(1)/usr/lib/libpth.so
+	ln -sf libpthsem.so.20.0.28  $(1)/usr/lib/libpth.so.20
+	ln -sf libpthsem.so.20.0.28  $(1)/usr/lib/libpth.so.20.0.28
 endef
 
 define Package/pthsem/install
 	$(INSTALL_DIR) $(1)/usr/lib
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpthsem.so.* $(1)/usr/lib/
+	ln -sf libpthsem.so.20.0.28  $(1)/usr/lib/libpth.so
+	ln -sf libpthsem.so.20.0.28  $(1)/usr/lib/libpth.so.20
+	ln -sf libpthsem.so.20.0.28  $(1)/usr/lib/libpth.so.20.0.28
 endef
 
 $(eval $(call BuildPackage,pthsem))
diff -urN pthsem.feeds.orig/packages/libs/pthsem/patches/001-enable_linux3.patch feeds/packages/libs/pthsem/patches/001-enable_linux3.patch
--- pthsem.feeds.orig/packages/libs/pthsem/patches/001-enable_linux3.patch	1970-01-01 01:00:00.000000000 +0100
+++ feeds/packages/libs/pthsem/patches/001-enable_linux3.patch	2013-11-21 22:15:57.399656151 +0100
@@ -0,0 +1,12 @@
+Index: pthsem-2.0.8/acinclude.m4
+===================================================================
+--- pthsem-2.0.8.orig/acinclude.m4	2009-10-27 08:49:25.000000000 +0100
++++ pthsem-2.0.8/acinclude.m4	2013-11-14 12:57:09.771322421 +0100
+@@ -892,6 +892,7 @@
+         case "x`uname -r`" in
+ changequote(, )dnl
+             x2.[23456789]* ) ;;
++            x3.* ) ;;
+ changequote([, ])
+             * ) braindead=yes ;;
+         esac
