diff -urN tasklet.package.orig/libs/tasklet/Makefile package/libs/tasklet/Makefile
--- tasklet.package.orig/libs/tasklet/Makefile	1970-01-01 01:00:00.000000000 +0100
+++ package/libs/tasklet/Makefile	2013-12-16 15:25:10.278667172 +0100
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2013 Luca Dionisi aka Lukisi
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=tasklet
+PKG_VERSION:=0.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://download-mirror.savannah.gnu.org/releases/netsukuku
+PKG_MD5SUM:=23d2c30fabebd610898ea1f4e4de8b06
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/tasklet
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=Lightweight cooperative threads
+  URL:=https://savannah.nongnu.org/projects/netsukuku/
+  DEPENDS:=+glib2 +libgee-0.8 +pthsem
+endef
+
+define Package/tasklet/description
+  Tasklet is a library that allows your programs to spawn a number of light cooperative
+  microthreads. It is used by netsukuku.
+endef
+
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/usr/include
+	$(CP)   $(PKG_INSTALL_DIR)/usr/include/*.h \
+		$(1)/usr/include/
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP)   $(PKG_INSTALL_DIR)/usr/lib/libtasklet.{a,so*} \
+		$(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(CP) \
+		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/tasklet.pc \
+		$(1)/usr/lib/pkgconfig/
+endef
+
+define Package/tasklet/install
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libtasklet.so* $(1)/usr/lib/
+endef
+
+$(eval $(call BuildPackage,tasklet))
